content/about/feature-stages/index.md

```diff
@@ -74,12 +78,10 @@ Below is our list of existing features and their current phases. This informatio
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
 | [VM: Service Credential Distribution](/docs/concepts/security/#pki)         | Beta
 | [Mutual TLS Migration](/docs/tasks/security/mtls-migration)    | Beta
-| [Authentication policy](/docs/concepts/security/#authentication-policies)  | Alpha
+| [Authorization (RBAC)](/docs/concepts/security/#authorization)   | Beta
```

Comment:
do we want to call this beta?  I feel the new API is under reviewed last week.

Reply:
Set it back to Alpha and added a note that the impl could be considered Beta but the API is still subject to change like an Alpha